miniapp/USERGUIDE.md

```diff
@@ -481,6 +482,12 @@ The following user data types are supported. If your App does not support a cert
 - Access Token: OAuth 1.0 token including token data and expiration date. Your App will be provided with the ID of the mini app and [AccessTokenScope]((api/com.rakuten.tech.mobile.miniapp.permission/-access-token-scope)) which is requesting the Access Token, so you should verify that this mini app is allowed to use the access token. See [UserInfoBridgeDispatcher.getAccessToken](api/com.rakuten.tech.mobile.miniapp.js.userinfo/-user-info-bridge-dispatcher/get-access-token.html)
 
 ### Ads Integration
```

Comment:
I think we should also document here which version of AdMob SDK is currently used by the `admob-latest` module.